test/test.go

```diff
@@ -274,6 +281,17 @@ func findTappable(c fyne.Canvas, pos fyne.Position) (o fyne.CanvasObject, p fyne
 	return
 }
 
+func findMouseable(c fyne.Canvas, pos fyne.Position) (o fyne.CanvasObject, p fyne.Position) {
+	matches := func(object fyne.CanvasObject) bool {
+		if _, ok := object.(desktop.Mouseable); ok {
+			return true
+		}
+		return false
```

Comment:
```suggestion
		_, ok := object.(desktop.Mouseable)
		return ok
```
You should be able to shorten that down to the following.